splunk_connect_for_snmp_poller/manager/task_utilities.py

```diff
@@ -126,104 +133,67 @@ def get_translated_string(mib_server_url, varBinds):
     return result, is_metric
 
 
-def mib_string_handler(
-    snmp_engine,
-    auth_data,
-    context_data,
-    host,
-    port,
-    mib_string,
-    mib_server_url,
-    index,
-    otel_logs_url,
-    otel_metrics_url,
-    one_time_flag,
-):
+def mib_string_handler(mib_list: list) -> VarbindCollection:
     """
     Perform the SNMP Get for mib-name/string, where mib string is a list
     1) case 1: with mib index - consider it as a single oid -> snmpget
     e.g. ['SNMPv2-MIB', 'sysUpTime',0] (syntax -> [<mib_file_name>, <mib_name/string>, <min_index>])
-    execute snmpget to query info correlated to this specific mib-name/string
 
-    2) case 2: without mib index - consider it as a oid with * -> snmpwalk
+    2) case 2: without mib index - consider it as a oid with * -> snmpbulk
```

Comment:
I think this is still snmp walk but implemented using getbulk not get command.
Actually this is equivalent of shell command snmpbulkwalk.

Reply:
Changed it.